src/components/PopUps/PopUpMessageBox.tsx

```diff
@@ -113,6 +114,16 @@ const PopUpMessageBox = ({message}: PopUpMessageProps) => {
                 <Typography level={"body-sm"}>
                     {messageStr}
                 </Typography>
+                {button && (
+                    <Button
+                        className={"pop-up-message-box-callback-button"}
+                        color={color}
+                        variant={"soft"}
+                        onClick={button.callback}
+                    >
+                        {button.title}
+                    </Button>
```

Comment:
Following the type change in `src/typings/notifications.ts`, how about
```suggestion
                    {...primaryAction}/>
```